getnacelle/nacelle-sanity-content-studio
# [LD-1099] Upgrade packages

### WHY are these changes introduced?

Addresses [LD-1099](https://nacelle.atlassian.net/browse/LD-1099)

> Improve @nacelle/sanity-content-studio



### WHAT is this pull request doing?

- Updates packages to take advantage of [the latest Sanity features, bug fixes, and security updates](https://www.sanity.io/changelog)
- Adds a new `upgrade` script to streamline the upgrade process in the future

## 

### How to Test

1. check out the `LD-1099-update-sanity-content-studio` branch
2. `npm i`
3. `npm run test`
4. `npm run start`
